packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1158,6 +1174,15 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
             this.deltaManager.on("op", this.onOp);
         }
 
+        // logging hardware telemetry
+        const { deviceMemory, hardwareConcurrency } = getDeviceSpec();
```

Comment:
There is no need to expand properties here. You can just do 
 logger.sendTelemetryEvent({
            eventName:"DeviceSpec",
            ...getDeviceSpec(),
        });

That way you automatically get any new props added in the future.